### ISSUE TITLE
Remove outdated and redundant SC4-1-1-accesskey draft rule

### ISSUE THREAD

**Issue by EmmaJP** [auto-wcag/auto-wcag#329]
https://github.com/auto-wcag/auto-wcag/blob/master/_drafts/outdated/SC4-1-1-accesskey.md is a redundant draft rule, as WCAG removed the failure technique that is was based on.

The file can be deleted.

**Comment by EmmaJP** [comment]
Resolved by pull request #332
